src/pages/examples.tsx

```diff
@@ -90,6 +99,9 @@ function reducer(state: State, action: Action) {
 				...state,
 				plugins: state.plugins.filter((p) => p !== action.value),
 			};
+		case 'enableGoldenRetriever': {
```

Comment:
I thought plugins are for the UI plugins. I tried to put goldenretriever inside the plugins array but it crashes. I think because goldenretriever is not a ui plugin and shouldnt be sent to <Dashboard plugins